Readme.md

```diff
@@ -47,3 +47,19 @@ Where:
 - `token` - the token to access the Qase API
 - `testrun-id` - the id of the test run in Qase
 - `output` - the file to save the expected results
+
+## How to use github action
+
+To use this tool in the github action, you need to create a workflow file in the `.github/workflows` directory and add
+the following step. For example:
+
+```yaml
+- name: Run Reporter Validator
+  uses: qase-tms/reporters-validator@v1
```

Comment:
using `@v1` will require us to put a tag `v1`, and then update this version explicitly. Let's maybe use `@master` for now? We can change from branch to tag when we have the first stable version